openkfw/TruBudget
# 2105: Refactor workflow action styles

### Checklist




### Description
- see #2105 

### How to test

1. login as user
2. go to subproject then to workflow action list
3. see new styles are in place
4. ...

Closes #2105
